### PR TITLE
Update the instructions for compatibility with commander v2

### PR DESCRIPTION
**Pull request by farshidtz**
- Related to #26
- Related to https://github.com/canonical/matter-pi-gpio-commander/pull/76

### CODE REVIEW

File: tutorial/pi-gpio-commander.md [hunk 1]
```diff
@@ -25,26 +23,61 @@ SSH to the Raspberry Pi and install the snap:
...
+
+
+### Configure the GPIO
+Set the GPIO pin/line to 4:
 ```bash
 sudo snap set matter-pi-gpio-commander gpio=4
 ```
 
-Grant the snap access to the GPIO memory, needed by this application:
+Apart from the GPIO pin, you may need to configure the GPIO chip.
```
Comment (locnnil) [inline tutorial/pi-gpio-commander.md:55]:
```suggestion
Apart from the GPIO pin, the GPIO chip may need to be configured.
```

File: tutorial/pi-gpio-commander.md [hunk 2]
```diff
@@ -111,15 +144,16 @@ sudo snap set matter-pi-gpio-commander args="--thread"
...
 
 To allow communication with the [OTBR Snap] for Thread management, connect the following interface:
 
 ```bash
 sudo snap connect matter-pi-gpio-commander:otbr-dbus-wpan0 \
                   openthread-border-router:dbus-wpan0
-```      
+```
 
+You may refer to [this guide](../how-to/otbr-on-ubuntu) setting up OTBR on Ubuntu.
```
Comment (locnnil) [inline tutorial/pi-gpio-commander.md:156]:
```suggestion
Refer to [this guide](../how-to/otbr-on-ubuntu) for setting up OTBR on Ubuntu.
```

**Review by locnnil — COMMENTED** [review]
I have some suggestions to change the tone of the text to be more impersonal. Please let me know if you agree. Thank you.

File: tutorial/pi-gpio-commander.md (hunk 2 above)
Comment (farshidtz) [inline tutorial/pi-gpio-commander.md:156]:
```suggestion
You may refer to [this guide](../how-to/otbr-on-ubuntu) for setting up OTBR on Ubuntu.
```

**Comment by farshidtz** [comment]
> I have some suggestions to change the tone of the text to be more impersonal. Please let me know if you agree. Thank you.

This is a tutorial, teaching the reader along the lines. There are steps that we expect the reader to perform with us (we), and others which the reader may need to do alone (you).

**Review by locnnil — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**